src/room/AppSelectionModal.tsx

```diff
@@ -62,7 +62,7 @@ export const AppSelectionModal: FC<Props> = ({ roomId }) => {
       return params;
     });
 
-    const result = new URL("element://call");
+    const result = new URL("io.element.call:/call");
```

Comment:
In the new schema, the path is supposed to be empty

```suggestion
    const result = new URL("io.element.call:/");
```